### PR TITLE
Update shopify.extension.toml

### PR DESCRIPTION
**Pull request by sjanczy2**

### CODE REVIEW

File: extensions/custom-banner/shopify.extension.toml [hunk 1]
```diff
@@ -3,22 +3,20 @@
...
 # and the file that contains your extension’s source code. Learn more:
-# https://shopify.dev/docs/api/checkout-ui-extensions/unstable/targets-overview
+# https://shopify.dev/docs/api/checkout-ui-extensions/unstable/extension-targets-overview
 
-#[START custom-banner.config-ext-point]
 [[extensions.targeting]]
-target = "purchase.checkout.block.render"
-module = "./src/Checkout.js"
-#[END custom-banner.config-ext-point]
+module = "./src/suggest.js"
```
Comment (hheyhhay) [inline extensions/custom-banner/shopify.extension.toml]: this `suggest.js` file is ONLY for the address autocomplete. This needs to stay the same as `./src/Checkout.js`

File: extensions/custom-banner/shopify.extension.toml [hunk 2]
```diff
@@ -3,22 +3,20 @@
 
 # The version of APIs your extension will receive. Learn more:
 # https://shopify.dev/docs/api/usage/versioning
-api_version = "2023-10"
+api_version = "2024-07"
 
 [[extensions]]
+name = "example-checkout-address--js"
```
Comment (hheyhhay) [inline extensions/custom-banner/shopify.extension.toml]: Same here. This copy and paste is from the address autocomplete file.

File: extensions/custom-banner/shopify.extension.toml (hunk 1 above)
Comment (hheyhhay) [reply extensions/custom-banner/shopify.extension.toml]:
```suggestion
module = "./src/Checkout.js"
```

File: extensions/custom-banner/shopify.extension.toml [hunk 3]
```diff
@@ -30,20 +28,22 @@ api_access = true
 # https://shopify.dev/docs/api/checkout-ui-extensions/unstable/configuration#network-access
 network_access = true
 
-# Loads metafields on checkout resources, including the cart,
-# products, customers, and more. Learn more:
-# https://shopify.dev/docs/api/checkout-ui-extensions/unstable/configuration#metafields
+api_version = "2024-07"
+
+[[extensions]]
```
Comment (hheyhhay) [inline extensions/custom-banner/shopify.extension.toml]: I think you want to keep this, delete whats above.

**Comment by sjanczy2** [comment]
@hheyhhay I copy pasted too much from auto complete, I'll check to see if I messed that up anywhere else

**Review by hheyhhay — APPROVED** [review]
LGTM!

**Pull request merged**